segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageConfig.java

```diff
@@ -33,6 +33,7 @@
     public static final Property<Integer> MAX_INDEXED_SEGMENTS = Property.named("readindex.segments.max", 1024);
     public static final Property<Integer> MAX_INDEXED_CHUNKS_PER_SEGMENTS = Property.named("readindex.chunksPerSegment.max", 1024);
     public static final Property<Integer> MAX_INDEXED_CHUNKS = Property.named("readindex.chunks.max", 16 * 1024);
+    public static final Property<Long> READ_INDEX_BLOCK_SIZE = Property.named("readindex.block.size", 64 * 1024 * 1024L);
```

Comment:
This should probably be sync-ed up with the Segment Read Index read alignment, which is defaulted to 1MB. 

If the upstream code only requests data in 1MB increments, how is 64MB block size helpful here?

Reply:
The search for chunk starts with 64MB  block that is floor to the give offset. 
There is some runtime overhead in that kind of scan in metadata. 
I think we can tune that. The block size does not need to match the 1MB increments but ideally should.
But there is also overhead on number of index entry.

Reply:
Fixed